### PR TITLE
ytdl: Adapt to new subtitles structure

### PR DESCRIPTION
**Pull request by jaimeMF**
Will be needed when I merge https://github.com/rg3/youtube-dl/pull/4964.

### CODE REVIEW

File: player/lua/ytdl_hook.lua [hunk 1]
```diff
@@ -3,7 +3,7 @@ local msg = require 'mp.msg'
 
 local ytdl = {
     path = "youtube-dl",
-    minver = "2014.11.26",
+    minver = "2014.11.26", -- TODO: update to version with the new subtitles system?
```
Comment (jaimeMF) [inline player/lua/ytdl_hook.lua]: Videos can be played with the old version, but there won't be subtitles. I can keep it unchanged if you want.

**Comment by ChrisK2** [comment]
I say bump the minver, as the change will break previously working functionality.

**Comment by wm4** [comment]
Just wondering, what sites will give us a direct subtitle URL?

**Comment by wm4** [comment]
And I suppose this also removes ``[ytdl_hook] WARNING: video doesn't have subtitles``.

**Comment by jaimeMF** [comment]
> Just wondering, what sites will give us a direct subtitle URL?

Youtube, ted and vimeo for example. Test urls: https://youtube.com/watch?v=QRS8MkLhQmM, http://www.ted.com/talks/dan_dennett_on_our_consciousness.html or http://vimeo.com/76979871.

> And I suppose this also removes [ytdl_hook] WARNING: video doesn't have subtitles.

No, you can use the `--no-warnings` option, since I don't think you need them.

**Comment by jaimeMF** [comment]
It's available in youtube-dl 2015.02.23.1.

**Comment by wm4** [comment]
Merged, thanks.

**Pull request closed without merging**